File: OnlineShopping/OnlineShopping/Models/MyOrderModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OnlineShopping.Models
{
    public class MyOrderModel
    {
        public int OrderID { get; set; }
        public int UserID { get; set; }
        public double OrderTotal { get; set; }
        public DateTime OrderDate { get; set; }
        public string Image { get; set; }
        public List<CartModel> CartModel { get; set; }
    }
}

File: prjonlineshopping/src/app/models/MyOrder.model.ts
import { CartModel } from './Cart.model';

export class MyOrderModel {
    constructor() {
        this.OrderID = null;
        this.UserID = null;
        this.OrderTotal = 0;
        this.OrderDate = null;
        this.CartModel = new Array<CartModel>();
    }
    OrderID: number;
    UserID: number;
    OrderTotal: number;
    OrderDate: Date;
    CartModel: CartModel[];
}
